check/bazel

```diff
@@ -14,3 +14,8 @@ cd $(git rev-parse --show-toplevel)
 bazel build //cirq/api/google/v1:operations_proto
 bazel build //cirq/api/google/v1:params_proto
 bazel build //cirq/api/google/v1:program_proto
+
+bazel build //cirq/api/google/v2:metrics_proto
```

Comment:
Are we able to build these on travis to verify they work, or is it still extremely difficult to get the proto compiler installed?

Reply:
We used to: https://github.com/quantumlib/Cirq/commit/4e37ea18ffdd9e2d9c75a282ae89eee4297ea580,

Until we reverted it due to a JDK download problem: https://github.com/quantumlib/Cirq/issues/83